.eslintrc

```diff
@@ -72,7 +72,7 @@
     "babel/generator-star-spacing": ["error", { before: false, after: true }],
 
     "react/display-name": ["error", { ignoreTranspilerName: true }],
-
+    "react/prefer-stateless-function": ["off"],
```

Comment:
I don't understand -- my reasoning for (re)removing this is turning stateless function components into React components is tedious when needed, and there's no perf overhead for doing so